src/compiler/python_generator.cc

```diff
@@ -303,12 +344,13 @@ bool PrintBetaServerFactory(const grpc::string& package_qualified_service_name,
                                    &output_message_module_and_class)) {
         return false;
       }
+      grpc::string method_name = ToSnakeCase(method->name());
       method_implementation_constructors.insert(
-          make_pair(method->name(), method_implementation_constructor));
+          make_pair(method_name, method_implementation_constructor));
       input_message_modules_and_classes.insert(
```

Comment:
I don't think we want to mangle the serialize/deserialize names, on the wire those should be the raw proto names.